### ISSUE TITLE
balena env fails with device not found if starts with 0

### ISSUE THREAD

**Issue by Davidnet** [balena-io/balena-cli#1150]
balena version: 9.15.2

If I do: 

```bash
$ balena env add TEST_CLI 1 --device 0XXXXX
BalenaDeviceNotFound XXXX (no zero) 
```

but with long id it works.

**Comment by Davidnet** [comment]
Related to https://github.com/balena-io/balena-cli/issues/1108 but still in 9.15.2

**Comment by Davidnet** [comment]
And yes my last UUID consists of only dec numbers.

**Comment by pdcastro** [comment]
@Davidnet, thank you for reporting this issue. I'll try to get a fix pull request up for review later today.

**Comment by Davidnet** [comment]
Is this resolved, I cannot find them on the Changelog?

**Comment by pdcastro** [comment]
@Davidnet, apologies for the delay. At the time I chose to bundle this issue in an ongoing pull request that transitions the `balena env` command from the [Capitano](https://github.com/balena-io/capitano/) framework to the [oclif](https://oclif.io/docs/introduction) framework (because oclif does not automatically try to convert arguments to integers, which is what causes the leading zero to be discarded), but some complications arose and it got delayed.

Could you confirm if this issue is blocking you in any way? I have found that the `balena env add` command accepts both short 7-character device UUIDs and full 32-character UUIDs, so one way to get around the error you were getting is to use the 32-character UUID (because it will certainly contain some non-numeric characters and the leading zero won't be discarded). But if this workaround poses complications to you, I will put together some temporary "hacky" fix. Thanks!

**Comment by pdcastro** [comment]
Fixed in CLI v11, back in May. The `env rename` command was fixed in CLI v11.11.1, back in September.
